pkg/api/types.go

```diff
@@ -283,6 +283,8 @@ type LivenessProbe struct {
 	Exec *ExecAction `json:"exec,omitempty"`
 	// Length of time before health checking is activated.  In seconds.
 	InitialDelaySeconds int64 `json:"initialDelaySeconds,omitempty"`
+	// Length of time before health checking times out.  In seconds.
+	TimeoutSecs int64 `json:"timeoutSecs,omitempty"`
```

Comment:
We're not going to probe, kill, and restart a container in less than a second. That could lead to all sorts of problems. Seconds seems fine.

Seconds should be spelled out.